app/views/posts/index.html.erb

```diff
@@ -1 +1,32 @@
-<h1>User post</h1>
+<section>
+ <div class='card_container'>
+      <img alt='This is an image' src='<%=@user.photo%>' class='user_image'/>
+      <div class='user-info'>
+        <p><%= link_to @user.name, @user, class: 'user-name' %></p>
+        <p>Number of posts: <%= @user.posts_counter%></p>
+      </div>
+    </div>
+</section>
+
+<section>
+  <% @posts.each do |post| %>
+    <article class='post-card'>
+      <div>
+        <a href='posts/<%=post.id%>'><p><%= post.title %></a></p>
+        <p maxlength= 150><%= post.text %></p>
+      </div>
```

Comment:
* Kindly use `post.recent_comments`  method to render 5 most recent comments, you have this methods created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md). to render the user's 5 most recent posts. [as mentioned in the project requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.)

Reply:
Hi @Davidosky007 tanks a lot for your review, I added next to that's lines of code, a conditional to show the recent comments if the post has.